<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QPICkit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/img.qrc">
    <normaloff>:/PICkit2.png</normaloff>:/PICkit2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPIC">
       <attribute name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/processor.png</normaloff>:/processor.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;PIC</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="hexFileButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;HEX File...</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/browse.png</normaloff>:/browse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="hexFileLineEdit">
          <property name="font">
           <font>
            <family>Noto Sans</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="programButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Program</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/burn.png</normaloff>:/burn.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="detectButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Detect PIC</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/processor.png</normaloff>:/processor.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="verifyButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Verify</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/verify.png</normaloff>:/verify.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="blankCheckButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Blank Check</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/refresh.png</normaloff>:/refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="readButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Read</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/read.png</normaloff>:/read.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="eraseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Erase</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/eraser.png</normaloff>:/eraser.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="aboutButton">
          <property name="text">
           <string>&amp;About</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/information.png</normaloff>:/information.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/log.png</normaloff>:/log.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Log viewer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="logTextArea">
          <property name="font">
           <font>
            <family>Noto Mono</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPICkit2">
       <attribute name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/PICkit2.png</normaloff>:/PICkit2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;PICKit2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QPushButton" name="setNewIDButton">
          <property name="text">
           <string>Set new ID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="setNewIDLineEdit">
          <property name="text">
           <string>PICkit2</string>
          </property>
          <property name="maxLength">
           <number>14</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="pickitInfoTextArea">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Noto Mono</family>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="detectPICkitButton">
          <property name="text">
           <string>Detect programmer</string>
          </property>
          <property name="icon">
           <iconset resource="img/img.qrc">
            <normaloff>:/PICkit2.png</normaloff>:/PICkit2.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="icon">
        <iconset resource="img/img.qrc">
         <normaloff>:/configuration.png</normaloff>:/configuration.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Settings</string>
       </attribute>
       <widget class="QRadioButton" name="pk2RadioButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;PICkit2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="appRadioButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;ArdPicProg</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QLabel" name="programmerLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Programmer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
